api/cmd/portainer/main.go

```diff
@@ -80,12 +81,8 @@ func initDataStore(dataStorePath string, fileService portainer.FileService) port
 func initComposeStackManager(assetsPath string, dataStorePath string, reverseTunnelService portainer.ReverseTunnelService, proxyManager *proxy.Manager) portainer.ComposeStackManager {
 	composeWrapper, err := exec.NewComposeStackManager(assetsPath, dataStorePath, proxyManager)
 	if err != nil {
-		if err == wrapper.ErrBinaryNotFound {
-			log.Printf("[INFO] [message: docker-compose binary not found, falling back to libcompose]")
-			return libcompose.NewComposeStackManager(dataStorePath, reverseTunnelService)
-		}
-
-		log.Fatalf("failed initalizing compose stack manager; err=%s", err)
+		log.Printf("[INFO] [main,compose] [message: falling-back to libcompose] [error: %s]", err)
```

Comment:
That's how this function looks in develop EE.
Besides, that implementation makes more sense to me: fallback to libcompose in case we couldn't initialise stack manager so that the app could still be mostly functional instead of completely crashing.

Reply:
:+1: